templates/_components/achievement_and_image.html.twig

```diff
@@ -0,0 +1,30 @@
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-quarter govuk-!-width-one-quarter" style="position: relative; height: 200px;">
```

Comment:
Please remove inline styles where possible and put them in an appropriate SCSS file.

Using pixel values for heights is also not recommended. I'm not sure on the precise context here, but this is likely to cause problems responsively (I also note that the height of this div is 50 pixels less than the height of the image that it contains).

I think it might be best for us to have a call to go through what you are trying to achieve here, and then maybe we can go through the best way to do it with CSS.

Just in general though, it is best to avoid PX and instead use REM or EM units (REM is easier to work with) for accessibility reasons.

(this is a very opinionated article on the matter, but it makes mostly valid points: https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9)

There is a mixin (function) that you can use in SCSS that allows you to convert pixel values to rem values:

`govuk-px-to-rem(40)`

Generally I have used this when working on the site, as it makes the code easier to work with (pixel values are more widespread in my experience) and also uses rem values, so results in a better UX.